### ISSUE TITLE
Where did the source code go?

### ISSUE THREAD

**Issue by Xanderplayz18** [vlang/vbrowser#9]
It was all removed for some reason.

**Comment by medvednikov** [comment]
There was no source, the couple of files that were posted by  someone else were unrelated.

If you're interested to join the dev team and get access to the source, contact me via discord: discord.gg/vlang (vox channel)

**Comment by xandaaah** [comment]
Makes sense. I mean, you do need to polish up code before open-sourcing.

**Comment by mindplay-dk** [comment]
Is this still being built?

I wish there was a browser that people actually wanted to work on - something that didn't take 45 minutes to compile. 😆
